frontend/src/components/BudgetLineItems/BudgetLinesTable/BLIRow.jsx

```diff
@@ -136,7 +137,7 @@ const BLIRow = ({
                 className={`${addErrorClassIfNotFound(budgetLine?.can?.number, isReviewMode)} ${borderExpandedStyles}`}
                 style={bgExpandedStyles}
             >
-                {canLabel(budgetLine)}
+                {isBudgetLineObe ? ("None") : canLabel(budgetLine)}
```

Comment:
would we want to target other budget lines tables?

Reply:
I'm not aware of any others??? 🤔  This one was being changed because it was changed everywhere else and was the only holdout I'm aware of.